README.md

```diff
@@ -1,2 +1,152 @@
 # crypto [![Build Status](https://travis-ci.com/CodeChain-io/rust-codechain-crypto.svg?branch=master)](https://travis-ci.com/CodeChain-io/rust-codechain-crypto) [![License: AGPL v3](https://img.shields.io/badge/License-AGPL%20v3-blue.svg)](https://www.gnu.org/licenses/agpl-3.0)
 The crypto library used by [CodeChain](https://github.com/CodeChain-io/codechain).
+
+## Usage
```

Comment:
It seems that the commit message is wrong. You added a usage to the README.md file, not the file itself.

Reply:
Oh, sorry. I modified it to 'Update the README.md' : )